integration_tests/conftest.py

```diff
@@ -1,40 +1,67 @@
+import signal
+import sys
+from typing import List
 import pytest
 import subprocess
 import pathlib
 import os
 import time
 
+def spawn_process(command: List[str]) -> subprocess.Popen[bytes]:
```

Comment:
@guilefoylegaurav , it should just be `subprocess.Popen` instead of `subprocess.Popen[bytes]` here.